CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+# Contributing guidelines
+
+We adhere to the [Contributor Covenant Code of Conduct](#contributor-covenant-code-of-conduct) found below.
+
+### To add a new feature
+
+- fork to your private github
+- create a new branch with a name meaningful for the feature at hand
+- commit all your work on that branch as you are progressing through the feature
+- when ready to push, create a pull request to the `master` branch
+
+### CI notes
+
+For GitHub Actions, the CI needs to pull and push docker images to a repository. We strongly suggest using 
+[docker hub](https://hub.docker.com).
+Please note that the project will `create` and `push` images in the following repositories:
+
+- webserver
+- director
+- storage 
+- catalog
+- sidecar
+- api-gateway
+
```

Comment:
Again, try to avoid specifics that might change with time. Put yourself in the situation of somebody that comes in one year. What is the important message here? Where will I most probably find the CURRENT list of services ? Get as close as possible to the code and we can keep a good documentation for long time. 

Here for instance, i would point the location of the C config so the user in the repo so the reader can look at the details of the actions

I would also add FAQ at the end with specific issues/problems you encountered at the time e.g. I know you had lots of pain going through dockehub e.g.
FAQ: Your PR CI is failing to push/pull images from dockerhub.  ANSWER: Did you check secrets in settings? See CI-notes